pages/bare_metal_cloud/dedicated_servers/bring-your-own-linux/guide.en-us.md

```diff
@@ -0,0 +1,160 @@
+---
```

Comment:
To be translated in french (fr-fr) (documentation team will then add additional languages afterward (after the production date when they will have more time)
Otherwise, yes, full byolinux documentation on github will only remain in english